layouts/contact/list.html

```diff
@@ -53,12 +53,12 @@ <h1 >Have Any Questins Or Emergency Problem Contact With Us</h1>
                           </div>
                           <div class="col-xl-6">
                               <select class="contact__input contact__input-3 contact__input-4">
-                                  <option value="">Choose Problem</option>
-                                  <option value="">Option 1</option>
-                                  <option value="">Option 2</option>
-                                  <option value="">Option 3</option>
-                                  <option value="">Option 4</option>
-                                  <option value="">Option 5</option>
+                                  <option value="">{{.Params.chooseProblem.list1}}</option>
```

Comment:
every single thing you don't need to do dynamic its okay to be static over here i think so

Reply:
done